ega-charts/localega/templates/verify-conf.yaml

```diff
@@ -30,6 +30,10 @@ data:
     cacertfile = /etc/ega/ssl/{{ .Values.config.tls_ca_root_file }}
     certfile = /etc/ega/ssl/verify{{ .Values.config.tls_cert_ending }}
     keyfile = /etc/ega/ssl/verify{{ .Values.config.tls_key_ending }}
+    [c4gh_file]
+    loader_class = C4GHFileKey
+    passphrase = {{ .Values.secrets.ega_c4gh_passphrase }}
+    filepath = /etc/ega/c4gh/ega_key.c4gh.sec
```

Comment:
in the long term maybe have this as a parameter as well. We do have it like that for other values: certs, keys etc.

Reply:
The file name at least, the path is fixed from the mount point of the secret.